pkg/restapi/v1/util/auth.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package util
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/labstack/echo/v4"
+
+	"github.com/trustbloc/vcs/pkg/restapi/resterr"
+)
+
+func GetOrgIDFromOIDC(ctx echo.Context) (string, error) {
```

Comment:
I don't see other code in this PR calling this function.

Reply:
Maybe controller.go file is collapsed in the review section by default, so the "find" command does not work. I used it here https://github.com/trustbloc/vcs/pull/776/files#diff-227f4ba89c786112771c32162dc814dece16c31c38c131df102921d58f240619R90